protocols/virtualfund/virtualfund.go

```diff
@@ -523,15 +533,36 @@ func ConstructObjectiveFromMessage(m protocols.Message, myAddress types.Address,
 		if !ok {
 			return Objective{}, fmt.Errorf("could not find a left ledger channel between %v and %v", intermediary, bob)
 		}
+
+		leftC, _ = getTwoPartyConsensusLedger(intermediary, bob)
+		// if !ok {
+		// 	// todo: #420 handle, after consensus_channel lifecycle is defined & channels present
+		// 	return Objective{}, fmt.Errorf("could not find a left ledger channel between %v and %v", intermediary, bob)
+		// }
+
 	} else if myAddress == intermediary {
 		left, ok = getTwoPartyLedger(alice, intermediary)
 		if !ok {
 			return Objective{}, fmt.Errorf("could not find a left ledger channel between %v and %v", alice, intermediary)
 		}
+
+		leftC, _ = getTwoPartyConsensusLedger(alice, intermediary)
+		// if !ok {
+		// 	// todo: #420 handle, after consensus_channel lifecycle is defined & channels present
+		// 	return Objective{}, fmt.Errorf("could not find a left ledger channel between %v and %v", alice, intermediary)
+		// }
+
 		right, ok = getTwoPartyLedger(intermediary, bob)
 		if !ok {
 			return Objective{}, fmt.Errorf("could not find a right ledger channel between %v and %v", intermediary, bob)
 		}
+
+		rightC, _ = getTwoPartyConsensusLedger(intermediary, bob)
+		// if !ok {
+		// 	// todo: #420 handle, after consensus_channel lifecycle is defined & channels present
+		// 	return Objective{}, fmt.Errorf("could not find a right ledger channel between %v and %v", intermediary, bob)
+		// }
+
```

Comment:
I think we are very close to being able to uncomment this, so I would suggest we do not merge this until that is possible?